### DISCUSSION TITLE
How to enable agent_vs_human option?

### DISCUSSION THREAD

**Discussion by Eternity402** [opendilab/LightZero#305] [Q&A]
Firstly, Thank you for your great work.

I tried to enable agent_vs_human option on _tictactoe_muzero_bot_mode_config.py_, but it did not work.

```
tictactoe_muzero_config = dict(
    exp_name=f'data_muzero/tictactoe_muzero_bot-mode_ns{num_simulations}_upc{update_per_collect}_rer{reanalyze_ratio}_seed0',
    env=dict(
        battle_mode='play_with_bot_mode',
        collector_env_num=collector_env_num,
        evaluator_env_num=evaluator_env_num,
        n_evaluator_episode=evaluator_env_num,
        manager=dict(shared_memory=False, ),
        agent_vs_human=True, # I added this line
    ),
```

I guess it's due to multiple evaluator envs, but it did not resolved with evaluator_env_num=1 option.

Error is like:
```
Enter the row (1, 2, or 3, from up to bottom) to play for the player 2: EOF when reading a line # I added Exception print
Wrong input, try again 
```

How can I enable it?

**Comment by puyuan1996** [comment]
Hello! You can set [`agent_vs_human`](https://github.com/opendilab/LightZero/blob/main/zoo/board_games/tictactoe/entry/tictactoe_muzero_eval.py#L27) to `True` in this [file](https://github.com/opendilab/LightZero/blob/main/zoo/board_games/tictactoe/entry/tictactoe_muzero_eval.py). After making this change, you can directly run the script from the command line to play against the agent. Note that MuZero will use a randomly initialized policy and take the first move. Thank you for your interest in this project!

**Reply by Eternity402** [reply]
Thank you for your reply!

I have tried agent_vs_human option set as True, but not worked for me.

After I changed `env_manager=dict(type='base'),` <- this, I could play with my bot

Maybe it depends on local env setting/machine etc
